crates/http/src/wagi.rs

```diff
@@ -143,7 +143,13 @@ impl HttpExecutor for WagiHttpExecutor {
             .or_else(ignore_successful_proc_exit_trap)?;
         tracing::info!("Module execution complete");
 
-        util::compose_response(&stdout_buffer.take())
+        let stdout = stdout_buffer.take();
+        ensure!(
+            !stdout.is_empty(),
+            "nothing was written to stdout; are you using a WAGI component?"
```

Comment:
As a reader I think this might suggest to me I _should_ be using a WAGI component but am not.  Perhaps "This module is configured to use the WAGI protocol, but did not write to stdout. Check the executor line in the manifest."